cylc/flow/task_remote_mgr.py

```diff
@@ -241,7 +241,7 @@ def remote_tidy(self):
             cmd.append(get_remote_suite_run_dir(host, owner, self.suite))
             procs[(host, owner)] = (
                 cmd,
-                Popen(cmd, stdout=PIPE, stderr=PIPE, stdin=open(DEVNULL)))
+                Popen(cmd, stdout=PIPE, stderr=PIPE))
```

Comment:
We have this in 7.8.x, and it works in master too:

```suggestion
                Popen(cmd, stdout=PIPE, stderr=PIPE, stderr=open(os.devnull)))
```

So, the question is: what's wrong with `subprocess.DEVNULL`, and should we restore the `os.devnull` version?

Reply:
Huh, a quick look at the `subprocess.py` and `os.py` source code suggests that `os.devnull` is the path `/dev/null` and `subprocess.DEVNULL` is the integer constant `-3`.  Hence the `ValueError` I guess. So maybe we're just using it incorrectly.

Reply:
Python docs:
> subprocess.DEVNULL
>
>    Special value that can be used as the `stdin`, `stdout` or `stderr` argument to `Popen` and indicates that the special file `os.devnull` will be used.

Huh.

Reply:
I'm running Python 3.7.3

Reply:
OK, duh :grimacing: 

It should be `stdin=DEVNULL` not `stdin=open(DEVNULL)`.  Adjusting the PR branch now...